test/completion.js

```diff
@@ -29,28 +29,26 @@ describe('Completion', () => {
   });
 
   describe('Test completion STDOUT output', () => {
-    it('Returns the completion candidates for both options and installed generators', done => {
+    it.only('Returns the completion candidates for both options and installed generators', () => {
       const yocomplete = path.join(__dirname, '../lib/completion/index.js');
       const yo = path.join(__dirname, '../lib/cli');
 
       let cmd = 'export cmd="yo" && YO_TEST=true DEBUG="tabtab*" COMP_POINT="4" COMP_LINE="$cmd" COMP_CWORD="$cmd"';
```

Comment:
Here, you probably need to add a space at the end of the line, otherwise the cmd will look like something along these lines: 

```bash
... COMP_CWORD="$cmd"node ...
```